src/testup/file_reporter.rb

```diff
@@ -75,7 +75,10 @@ def report
 
   def record(result)
     super
-    io.puts "%s#%s = %.2f s = %s" % [result.class,
+    unless instance_variable_defined? :@has_klass
```

Comment:
The logic in these two files are duplicated. I think it would be ideal if this was put into a Mix-in module that could reuse the logic across both reporters. And along with a comment that explains why this is being done the way it is.

Reply:
I'll be looking at some Minitest stuff later.  Minitest 5.11.3 (current) does not have a Ruby version constraint, and I believe this PR is due to testup-2 using an old version (not 100% sure).

Should we/you just update the Minitest version used?

Reply:
I got an issue logged to look into the Minitest version. I'm spinning up on TestUp2 work, so I'll get back to this shortly.

I don't recall why a newer version of MiniTest didn't work previously. I might have never found the real cause of it. So will have to see how the latest version fares.